base/src/main/java/com/smartdevicelink/proxy/rpc/enums/AppCapabilityType.java

```diff
@@ -0,0 +1,23 @@
+package com.smartdevicelink.proxy.rpc.enums;
```

Comment:
This File is missing the copyright because the generator was not used to create this file, Can you try again to generate this file using the generator as the PR for the rpc_spec has changed since you last tried?